### DISCUSSION TITLE
Can't remotely control spotube

### DISCUSSION THREAD

**Discussion by laukamimura** [KRTirtho/spotube#1528] [Q&A]
Hi, I installed spotube on a pc, an Ubuntu laptop and an android phone. 
I activated remote control in all 3 devices, but i can only control the pc version with the android version.
I cannot control the android version with pc or linux.
I cannot control the PC version with the linux vesion.
I cannot control the linux vesion with the android or pc version.

**Comment by tigerkiller55891** [comment]
basically same issue here assuming Im trying to do it correctly since aside from logging into spotify they don't give any documentation on how it works.
